ci/Dockerfile

```diff
@@ -0,0 +1,6 @@
+# CI image for building with gcc and clang
+FROM alpine:latest
+
+RUN apk --no-cache add clang cmake make binutils musl-dev g++ boost-dev gtest gtest-dev gfortran python3-dev && \
```

Comment:
Do we really need `musl-dev` and `gfortran`? O_O

Reply:
`musl-dev`: alpine comes with musl libc, we need it since it also contains the runtime object files, needed for creation of elf files.

`gfortran` is needed for compiling pandas (or deps of pandas). Probably, we should consider making pandas optional for building and testing, then we could remove `gfortran`. Also pip3 installations would be obsolete, since the other python packages are part of the alpine distro.

Reply:
I see. Thanks! +2